src/core/Table/Table.tsx

```diff
@@ -562,9 +561,10 @@ export const Table = <
                 onBottomReached={onBottomReachedRef}
                 intersectionMargin={intersectionMargin}
                 state={state}
-                key={rowProps.key}
+                key={row.getRowProps().key}
                 onClick={onRowClickHandler}
                 subComponent={subComponent}
+                isRowDisabled={!!isRowDisabled?.(row.original)}
```

Comment:
I guess could call just `disabled` or `isDisabled`, cause its Row component, no need to go long, imo

Reply:
Renamed.